src/v/config/config_store.h

```diff
@@ -70,42 +70,55 @@ class config_store {
 
         for (auto const& node : root_node) {
             auto name = node.first.as<ss::sstring>();
-            auto found = _properties.find(name);
-            if (found == _properties.end()) {
-                found = _aliases.find(name);
-                if (found == _aliases.end()) {
-                    if (!ignore_missing.contains(name)) {
-                        throw std::invalid_argument(
-                          fmt::format("Unknown property {}", name));
-                    }
+            auto* prop = [&]() -> base_property* {
+                auto primary = _properties.find(name);
+                if (primary != _properties.end()) {
+                    return primary->second;
                 }
-            } else {
-                bool ok = false;
-                try {
-                    auto validation_err = found->second->validate(node.second);
-                    if (validation_err.has_value()) {
-                        errors[name] = fmt::format(
-                          "Validation error: {}",
-                          validation_err.value().error_message());
-                    }
-
-                    found->second->set_value(node.second);
-                    ok = true;
```

Comment:
> config_store bug about not considering aliases for bootstrap

please describe the problem that is fixed in this commit, and how it is fixed. it also looks like there is some code moved / refactored? if so, that could be a separate commit to make it easier to understand the change.

Reply:
done, splitted it apart. the first commit has the first fix, the following are some cosmetic change to bring it up smoothly at this stage